api/pkg/apps/webapp/cases.go

```diff
@@ -363,39 +377,35 @@ func (s *Server) NewCase(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
-func (s *Server) PutOrPostCase(req *http.Request, w http.ResponseWriter, ctx context.Context, kase *cms.Case) error {
+func (s *Server) PostCase(req *http.Request, w http.ResponseWriter, ctx context.Context, kase *cms.Case) (*cms.Case, error) {
```

Comment:
case with "k" ??  is this typo?

Reply:
hehehe I guess you might also have been pulling your hair out trying to figure out why you were getting really weird syntax errors until you remember that `case` is a keyword :P